### DISCUSSION TITLE
mysql deadlock

### DISCUSSION THREAD

**Discussion by qinyang666** [apache/airflow#14801] [Q&A]
I have started three schedulers in Airflow2.0.1 and set scheduler_heartbeat_sec=60. Then I get an error

![image](https://user-images.githubusercontent.com/22069999/111168237-d81cd680-85dc-11eb-9fc1-192354150bd2.png)

**Comment by turbaszek** [comment]
https://github.com/apache/airflow/issues/13668 if this still occurs we should reopen the issue or create new one
